src/App.tsx

```diff
@@ -1,92 +1,50 @@
-import React, { Component } from 'react';
-import { getI18n, I18nextProvider } from 'react-i18next';
+import React, { useEffect, useState } from 'react';
+import { BrowserRouter, HashRouter } from 'react-router-dom';
 
-import type { Config } from '#types/Config';
-import Router from '#src/containers/Router/Router';
-import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
 import QueryProvider from '#src/providers/QueryProvider';
-import { restoreWatchHistory } from '#src/stores/WatchHistoryController';
-import { initializeAccount } from '#src/stores/AccountController';
-import { initializeFavorites } from '#src/stores/FavoritesController';
-import { logDev } from '#src/utils/common';
-import { loadAndValidateConfig } from '#src/utils/configLoad';
-import { clearStoredConfig } from '#src/utils/configOverride';
-import { PersonalShelf } from '#src/enum/PersonalShelf';
-import initI18n from '#src/i18n/config';
-
 import '#src/screenMapping';
 import '#src/styles/main.scss';
+import initI18n from '#src/i18n/config';
+import Root from '#components/Root/Root';
+import { ErrorPageWithoutTranslation } from '#components/ErrorPage/ErrorPage';
+import LoadingOverlay from '#components/LoadingOverlay/LoadingOverlay';
 
 interface State {
-  error: Error | null;
   isLoading: boolean;
+  error?: Error;
 }
 
-class App extends Component {
-  public state: State = {
-    error: null,
-    isLoading: true,
-  };
+export default function App() {
+  const [i18nState, seti18nState] = useState<State>({ isLoading: true });
 
-  componentDidCatch(error: Error) {
-    this.setState({ error });
-  }
-
-  async initializeServices(config: Config) {
-    if (config?.integrations?.cleeng?.id) {
-      await initializeAccount();
-    }
-
-    // We only request favorites and continue_watching data if there is a corresponding item in the content section
-    // and a playlist in the features section.
-    // We first initialize the account otherwise if we have favorites saved as externalData and in a local storage the sections may blink
-    if (config.features?.continueWatchingList && config.content.some((el) => el.type === PersonalShelf.ContinueWatching)) {
-      await restoreWatchHistory();
-    }
+  useEffect(() => {
```

Comment:
I took it out because of this: https://react.i18next.com/latest/i18nextprovider
"You will only need to use the provider if you need to support multiple i18next instances"